scss/forms/_form-range.scss

```diff
@@ -43,10 +43,19 @@
     @include transition($form-range-thumb-transition);
     appearance: none;
 
+    // Boosted mod
+    &:hover {
+      border: $form-range-thumb-border;
+      @include gradient-bg($form-range-thumb-hover-bg);
+    }
+    // End mod
+
     &:active {
       cursor: grabbing; // Boosted mod
+      border: $form-range-thumb-active-border; // Boosted mod
```

Comment:
If the first comment is taken into account, this should evolve to `border-color` too. Keep the file up to date if your fine with this change!

Reply:
OK